### PR TITLE
MNT Add support for sminnee/phpunit fork

### PR DESCRIPTION
**Pull request by dnsl48**

### CODE REVIEW

File: src/Extension.php [hunk 1]
```diff
@@ -52,8 +53,26 @@ public function getConfigKey()
...
-        // PHPUnit
-        require_once BASE_PATH . '/vendor/phpunit/phpunit/src/Framework/Assert/Functions.php';
+        // Find PHPUnit assertion implementations
+
+        $found = false;
+
+        $options = [
+            BASE_PATH . '/vendor/phpunit/phpunit/src/Framework/Assert/Functions.php',
+            BASE_PATH . '/vendor/sminnee/phpunit/src/Framework/Assert/Functions.php',
+        ];
```
Comment (Cheddam) [inline src/Extension.php]: @dnsl48 Won't `sminnee/phpunit` always be installed now that it's an explicit requirement of this module?

Comment (dnsl48) [reply src/Extension.php]: Fair enough. Although, if we update composer.json in the future versions of behat-extension and would use e.g. phpunit 9 or something like that, this code will keep working transparently.

Comment (Cheddam) [reply src/Extension.php]: Okay, I'll settle for shifting `sminnee` up so it saves a loop in the meantime 😛

Comment (Cheddam) [reply src/Extension.php]:
```suggestion
        $options = [
            BASE_PATH . '/vendor/sminnee/phpunit/src/Framework/Assert/Functions.php',
            BASE_PATH . '/vendor/phpunit/phpunit/src/Framework/Assert/Functions.php',
        ];
```

**Review by Cheddam — APPROVED** [review]

**Pull request merged**